### DISCUSSION TITLE
Environment variable NGINX_MAX_UPLOAD is not working

### DISCUSSION THREAD

**Discussion by sujeetkv** [tiangolo/uwsgi-nginx-flask-docker#365] [Questions] [answered]
I have set `ENV NGINX_MAX_UPLOAD 8m` in my Dockerfile but it seems not working because I am not able to upload more than 1MB content. NGINX rejects request saying - Request Entity Too Large.

**Comment by r1n9w0rm** [comment]
I too have had this issue using Docker w/ AWS Elastic Beanstalk.  This is what helped me - https://stackoverflow.com/a/61943622

**Comment by guysoft** [comment, marked as answer]
If you are sitting behind a reverse nginx-proxy you also need to take that in to account.
User the custom hostnames to create a file that contains:

```
client_max_body_size 1g;
```

More on how to put that here: https://github.com/nginx-proxy/nginx-proxy#per-virtual_host-location-configuration
